providers/ns1/ns1Provider.go

```diff
@@ -199,6 +204,13 @@ func convert(zr *dns.ZoneRecord, domain string) ([]*models.RecordConfig, error)
 			if err := rec.SetTarget(ans); err != nil {
 				panic(fmt.Errorf("unparsable %s record received from ns1: %w", rtype, err))
 			}
+		case "CAA":
+			//dnscontrol expects quotes around multivalue CAA entries, API doesn't add them, so let's force them
+			x_ans := strings.SplitN(ans, " ", 3)
+			answer := fmt.Sprintf(`%s %s "%s"`, x_ans[0],x_ans[1],x_ans[2])
+			if err := rec.PopulateFromString(rtype, answer, domain); err != nil {
+				panic(fmt.Errorf("unparsable %s record received from ns1: %w", err))
```

Comment:
Don't panic.  Return an error.

Reply:
You're right, cleared all panics from the provider :)